### PR TITLE
Adding login styling for students and admins

### PR DESCRIPTION
**Pull request by charlesx2013**
Some examples:

image

image

@eric-ls pls

### CODE REVIEW

File: app/assets/stylesheets/pages/login_page.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
+.login-container {
+  min-height: 100vh;
+}
+
+.login-modal {
+  padding: 0 0 80px 0;
```
Comment (eric-ls) [inline app/assets/stylesheets/pages/login_page.scss]: if there's only one value, use `padding-left: 80px` instead

File: app/assets/stylesheets/partials/_layout.scss [hunk 2]
```diff
@@ -14,6 +14,8 @@ $md: 24px;
 $lg: 32px;
 $xl: 40px;
 $xxl: 54px;
+$xxxl: 60px;
+$xxxxl: 72px;
```
Comment (eric-ls) [inline app/assets/stylesheets/partials/_layout.scss]: uhh its not wrong but do you really want to be typing in `xxxxxl` lol

**Comment by eric-ls** [comment]
also why does one screen say "admin login" and another say "lets get started"? 
pls also space things out a bit (like move the "sign up" and "forgot password?" links a little below so people dont accidentally click on it

**Comment by charlesx2013** [comment]
:+1:

**Pull request merged**